ydb/docs/en/core/reference/kafka-api/index.md

```diff
@@ -0,0 +1,8 @@
+# Kafka API
+
+{{ ydb-short-name }} supports working with [topics](../../concepts/topic.md) using [the Kafka protocol version 3.4.0](https://kafka.apache.org/34/documentation.html). It allows to integrate {{ ydb-short-name }} with applications originally developed to work with [Apache Kafka](https://kafka.apache.org/).
+
+The Kafka API  documentation contains the following sections:
+
+* [Usage examples](examples.md)

```

Comment:
List order is different from the ToC. It looks weird when open this page together wit ToC.

Reply:
> List order is different from the ToC. It looks weird when open this page together wit ToC.
✅